docs/dataset.md

```diff
@@ -63,47 +81,107 @@ Tables that lists all systems along with their annotations.
+- `protein_lddt_qcov_max`: Global protein structural similarity multiplied by query system coverage, aggregated by max score across all pairs of protein chains or ligand chains.
+- `protein_lddt_qcov_weighted_max`: Global protein structural similarity multiplied by query system coverage, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+- `protein_lddt_qcov_weighted_sum`: Global protein structural similarity multiplied by query system coverage, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
+- `protein_lddt_weighted_max`: Local structural similarity between chains of two systems, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+- `protein_lddt_weighted_sum`: Local structural similarity between chains of two systems, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
+- `protein_qcov_weighted_sum`: Global protein query coverage, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
+- `protein_seqsim_max`: Global protein sequence similarity between components of two systems, aggregated by max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_qcov_max`:  Global  protein sequence similarity between components of two systems multiplied by query system coverage,  aggregated by max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_qcov_weighted_max`: Global  protein sequence similarity between components of two systems multiplied by query system coverage, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_qcov_weighted_sum`: Global  protein sequence similarity between components of two systems multiplied by query system coverage, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
+- `protein_seqsim_weighted_max`:  Global protein sequence similarity between components of two systems, aggregated by length-weighted max score across all pairs of protein chains or ligand chains.
+- `protein_seqsim_weighted_sum`: Global protein sequence similarity between components of two systems, aggregated by length-weighted sum of scores across mapped protein or ligand chains.
 
-:::{list-table} `ligands_per_system.parquet` columns
-:widths: 10 5 30
-:header-rows: 1
 
-*   - Name
-    - Type
-    - Description
-*   - system_id
-    - str
-    - The PLINDER system ID
-*   - component
-    - str
-    - The cluster ID of the cluster the system belongs to
-:::
 
 ### Splits (`splits/`)
 
-This directory contains:
+This directory contains split files and the configs used to generate them.
 
-- `plinder-pl50.parquet`: listing the split category for each system
-- ...
+- `split.parquet`: listing the split category for each system
+- `split.yaml`: the config used to generate the split
```

Comment:
Do we need to describe the content of the yaml file as well?

Reply:
I would say not really necessary for the user, more for developers.
What would be a good way to express this?

Reply:
As the file is quite short, I would propose to print an example file with a short comment describing the parameter, analogous to the directory listing:

```
sampling_cluster_metric: pli_qcov  # ...
sampling_cluster_threshold: 70     # ...
...
```